app/components/events/view/overview/event-apps.js

```diff
@@ -1,7 +1,17 @@
 import classic from 'ember-classic-decorator';
 import { classNames } from '@ember-decorators/component';
 import Component from '@ember/component';
+import { computed } from '@ember/object';
+import ENV from 'open-event-frontend/config/environment';
 
 @classic
 @classNames('ui', 'fluid', 'card')
-export default class EventApps extends Component {}
+export default class EventApps extends Component {
+    @computed('eventId')
+  get webAppGeneratorUrl() {
+    if (this.authManager.currentUser) {
+      return `${ENV.appGenerator.webHost}/?email=${this.authManager.currentUser.email}&apiendpoint=${ENV.APP.apiHost}/${ENV.APP.apiNamespace}/events/${this.eventId}`;
+    }
+    return '#';
```

Comment:
Why would you return a

Reply:
Shall we return `${ENV.appGenerator.webHost}/`?

Reply:
When the check is removed, it won't be needed